doc/lsst.obs.base/creating-an-obs-package.rst

```diff
@@ -0,0 +1,28 @@
+.. creating-an-obs-package:
+
+####################################
+How to create a new LSST obs package
+####################################
+
+These instructions describe how to create a package that allows the LSST Data Management Science Pipeline to read raw data from a new telescope and instrument.
+
+1. MetadataTranslator
+=====================
+
+Create a ``MetadataTranslator`` derived from `~astro_metadata_translator.translators.MetadataTranslator` or `~astro_metadata_translator.translators.FitsTranslator`.
+
+2. Formatter
+============
+
+Create a ``FitsRawFormatter`` derived from `~lsst.obs.base.FitsRawFormatterBase`.
+
+At minimum, override the abstract methods `~lsst.obs.base.FitsRawFormatterBase.makeVisitInfo` and `~lsst.obs.base.FitsRawFormatterBase.getDetector`.
```

Comment:
If a MetadataTranslator exists then we know exactly how to make a VisitInfo and no-one should need to write their own subclass. The obs_base implementation is exactly what you need: call the `observationInfo2visitInfo` static method on the `ObservationInfo`.